avocado/core/result.py

```diff
@@ -194,15 +193,16 @@ def __init__(self, job):
         super(HumanResult, self).__init__(job)
         self.log = logging.getLogger("avocado.app")
         self.__throbber = output.Throbber()
+        self._replay_source_job = getattr(job.args, "replay_sourcejob", None)
 
     def start_tests(self):
         """
         Called once before any tests are executed.
         """
         super(HumanResult, self).start_tests()
         self.log.info("JOB ID     : %s", self.job_unique_id)
-        if getattr(self.args, "replay_sourcejob", None):
-            self.log.info("SRC JOB ID : %s", self.args.replay_sourcejob)
+        if self._replay_source_job is not None:
+            self.log.info("SRC JOB ID : %s", self._replay_source_job)
```

Comment:
looks better, I'm wondering whether remote plugins are happy about this change. If yes, then let's go with this change...